src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,161 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.NoSuchElementException;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
```

Comment:
no empty line after class definition is needed

Reply:
Ok